README.md

```diff
@@ -1,21 +1,92 @@
 # dockstore-tool-star
+Versioned Docker image and CWL tools for the STAR Read Aligner
 
-Dockerized STAR and CWL tool definition for Dockstore.
+# Description
+This repository contains a dockerfile and the following CWL tools:
```

Comment:
Do you feel that this is a sufficient level of detail?

Reply:
Do I feel there is a sufficient level of detail in the description? Yes.